spec/install/gemfile/path_spec.rb

```diff
@@ -399,7 +433,7 @@
     end
 
     it "unlocks all gems when a child dependency gem is updated" do
-      build_lib "bar", "2.0", :path => lib_path("foo/bar")
+      build_lib "bar", "2.0", :path => lib_path("foo")
```

Comment:
These two changes are questionable.  Without them, this test fails on the should_be_installed for bar, trying to require bar, which can't be found.